.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -8,106 +8,108 @@ assignees: ''
 
 # v<TARGET_RELEASE>
 
-## Code freeze
-
-- [ ] 1. Update Enclave CC to use the latest commit from image-rs
-
-    * https://github.com/confidential-containers/enclave-cc/blob/main/src/enclave-agent/Cargo.toml
-      * Change the revision
-      * Run `cargo update -p image-rs`
-
-- [ ] 2. Update Kata Containers to use the latest commit from image-rs, attestation-agent and td-shim
-
-    * image-rs
-      * https://github.com/kata-containers/kata-containers/blob/CCv0/src/agent/Cargo.toml
-      * Change the revision
-      * Run `cargo update -p image-rs`
-    * attestation-agent and td-shim
-      * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
-        * Change the version
-
-- [ ] 3. Wait for kata-runtime-payload-ci to be successfully built
-    * After the previous PR is merged wait for the kata-runtime-payload-ci (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml) has completed, so the latest kata-runtime-payload-ci contains the changes
+## Overview
+
+The release process mainly follows from this dependency graph.
+
+```mermaid
+flowchart LR
+    Trustee --> Versions.yaml
+    Guest-Components --> Versions.yaml
+    Kata --> kustomization.yaml
+    Guest-Components .-> Client-tool
+    Guest-Components --> enclave-agent
+    enclave-cc --> kustomization.yaml
+    Guest-Components --> versions.yaml
+    Trustee --> versions.yaml
+    Kata --> versions.yaml
+
+    subgraph Kata
+        Versions.yaml
+    end
+    subgraph Guest-Components
+    end
+    subgraph Trustee
+        Client-tool
+    end
+    subgraph enclave-cc
+        enclave-agent
+    end
+    subgraph Operator
+        kustomization.yaml
+        reqs-deploy
+    end
+    subgraph cloud-api-adaptor
+        versions.yaml
+    end
+```
+
+Starting wth v0.9.0 the release process no longer involves centralized dependency management.
+In other words, when doing a CoCo release, we don't push the most recent versions of the subprojects
+into Kata and enclave-cc. Instead, dependencies should be updated during the normal process of development.
+Releases of most subprojects are now decoupled from releases of the CoCo project.
+
+## The Steps
+
+### Determine release builds
+
+Identify/create the bundles that we will relase for Kata and enclave-cc.
+
+- [ ] 1. :eyes: **Create enclave-cc release**
+
+    Enclave-cc does not have regular releases apart from CoCo, so we need to make one.
+    Make sure that the CI is green and then use the Github release tool to create a tag and release.
+    This should create a bundle [here](https://quay.io/repository/confidential-containers/runtime-payload?tab=tags).
```

Comment:
I was just sanity-checking that the release.yaml for enclave-cc does this, and I noticed it triggers when a tag is pushed [here](https://github.com/confidential-containers/enclave-cc/blob/main/.github/workflows/release.yaml#L5).
I wonder if the upcoming `v0.9.0-alpha0` tag will be an edge case that this won't trigger?
It might need an asterisk, e.g. see `v1.*` [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags)

Reply:
Hm that's a good point. One option is to tag enclave-cc as v0.9.0. In fact that might be optimal since the release of enclave-cc isn't an alpha like the release of coco/operator is.

Reply:
(there [aren't](https://github.com/confidential-containers/operator/blob/main/.github/workflows/docker-publish-on-tag.yaml#L13))

Reply:
> One option is to tag enclave-cc as v0.9.0. In fact that might be optimal since the release of enclave-cc isn't an alpha like the release of coco/operator is.
Ah, a v0.9.0 tag for enclave-cc sounds right. That solves the problem, too.